### ISSUE TITLE
Improve external grading documentation

### ISSUE THREAD

**Issue by nwalters512** [PrairieLearn/PrairieLearn#705]
Current documentation is relatively thorough but not that accessible - its attempt at completeness makes it kind of hard to parse. I need to develop more easy-to-digest documentation, possibly in the form of more focused pieces ("How do I develop locally?", "How do I add external grading to a course?", etc.) or video walkthroughs.

If anyone who's worked with external grading recently had any input into what content would have been or would be helpful, please comment!

**Comment by mwest1066** [comment]
There have been a lot of small tweaks to the external grading docs lately (e.g., #2399). The docs can always be improved, but I don't think we need to keep this issue open to track it.
